gym/scoreboard/__init__.py

```diff
@@ -706,6 +712,32 @@
 """,
     )
 
+# ple
+for id in sorted(['Catcher-v0', 'MonsterKong-v0', 'FlappyBird-v0', 'PixelCopter-v0', 'PuckWorld-v0', 'RaycastMaze-v0', 'Snake-v0', 'WaterWorld-v0']):
+    try:
+        split = id.split("-")
+        game = split[0]
+        if len(split) == 2:
+            ob_type = 'image'
+        else:
+            ob_type = 'ram'
+    except ValueError as e:
+        raise ValueError('{}: id={}'.format(e, id))
+    ob_desc = ram_desc if ob_type == "ram" else image_desc
+    add_task(
+        id=id,
+        group='ple',
+        contributor='lusob',
+        summary="Maximize score in the game %(game)s, with %(ob_type)s as input"%dict(game=game, ob_type="RAM" if ob_type=="ram" else "screen images"),
+        description="""\
+Maximize your score in the pygame game %(game)s.
```

Comment:
You should include some instructions here so people know how to obtain your environments.